include/aspect/assembly.h

```diff
@@ -398,8 +398,8 @@ namespace aspect
         boost::signals2::signal<void (const typename DoFHandler<dim>::active_cell_iterator &,
                                       const double,
                                       const bool,
-                                      internal::Assembly::Scratch::StokesSystem<dim>       &,
-                                      internal::Assembly::CopyData::StokesSystem<dim>      &)> local_assemble_stokes_system;
+                                      internal::Assembly::Scratch::StokesSystem<dim> &,
+                                      internal::Assembly::CopyData::StokesSystem<dim> &)> local_assemble_stokes_system;
```

Comment:
here, and in line 427, this is an unrelated change. can you restore the original indentation?

(I say these things not because I'm super pedantic -- which is also true -- but primarily because with many others developing code in this area right now, anyone who changes lines and gets them merged may cause merge conflicts for others who also made changes in this area but did not get them merged yet. It's therefore always useful to avoid unnecessary changes in unrelated areas to avoid stepping on other developers' feet.)